pyproject.toml

```diff
@@ -95,8 +93,10 @@ flask = "^2.3.3"
 httpx = { extras = ["cli"], version = "^0.24.1" }
 mypy = "^1.0.0"
 pytest-split = "^0.8.1"
-requests-mock = "^1.11.0"
-types-requests = "2.31.0.2"
+requests-mock = ">=1.11.0"
+types-requests = ">=2.31.0.2"
+uvicorn = ">=0.24.0"
+uvloop = ">=0.17.0"
```

Comment:
We have above in `[tool.poetry.group.builder.dependencies]` exact versions pinned to `tenacity = "^8.0.1"` and `uvicorn = "^0.24.0"`.

It seems a bit strange to specify the same package multiple times but with different version requirements. Would it be possible to pick one version (the lower bound preferably) and put it into a "shared" group instead?